homeassistant/components/unifiprotect/sensor.py

```diff
@@ -252,6 +261,27 @@ def _get_alarm_sound(obj: Sensor) -> str:
         ufp_value_fn=_get_alarm_sound,
         ufp_enabled="is_alarm_sensor_enabled",
     ),
+    ProtectSensorEntityDescription(
+        key="door_last_trip_time",
+        name="Last Open",
+        device_class=SensorDeviceClass.TIMESTAMP,
+        ufp_value="open_status_changed_at",
+        entity_registry_enabled_default=True,
```

Comment:
This should be `False` as `True` is the default.

Reply:
Oops.  Will fix when I get back home from dinner

Reply:
d37544f04b1cb1d05260a940d6732e2b6962151c